### PR TITLE
cloud: move Import Data to ...

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
 1. In the TiDB Cloud Console, get the TiDB Cloud account ID and external ID of the target TiDB cluster.
 
     1. In the TiDB Cloud Console, choose your target project, and navigate to the **Clusters** page.
-    2. Locate your target cluster, and click **Import Data** in the upper-right corner of the cluster area. The **Data Import Task** page is displayed.
+    2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import Task** page is displayed.
```
Comment (hfxsd) [inline tidb-cloud/config-s3-and-gcs-access.md:20]:
```suggestion
    2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
...
 
         > **Tip:**
         >
-        > Alternatively, you can also click the name of your cluster on the **Clusters** page and click **Import Data** in the upper-right corner.
+        > Alternatively, you can also click the name of your cluster on the **Clusters** page and click **Import Data** in the **Import** area.
 
     3. On the **Data Import Task** page, click **Show AWS IAM policy settings** to get the TiDB Cloud Account ID and TiDB Cloud External ID. Take a note of these IDs for later use.
```
Comment (hfxsd) [inline tidb-cloud/config-s3-and-gcs-access.md:26]:
```suggestion
    3. On the **Data Import** page, click **Show AWS IAM policy settings** to get the TiDB Cloud Account ID and TiDB Cloud External ID. Take a note of these IDs for later use.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -120,11 +120,11 @@ To create a Dedicated Tier cluster, take the following steps:
 We provide Capital Bikeshare sample data for you to easily import data and run sample queries.
 
 1. Navigate to the **Clusters** page.
-2. In the area of your newly created cluster, click **Import Data** in the upper-right corner. The **Data Import Task** page is displayed.
+2. In the area of your newly created cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import Task** page is displayed.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-quickstart.md:123]:
```suggestion
2. In the area of your newly created cluster, click **...** in the upper-right corner, and select **Import Data**. The **Data Import Task** page is displayed.
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 4]
```diff
@@ -105,7 +105,7 @@ To allow TiDB cloud to access the source data in your GCS bucket, you need to co
 
 1. Get the Google Cloud Service Account ID of the target TiDB cluster.
 
-    1. In the TiDB Cloud Admin console, choose a target project and a target cluster deployed on the Google Cloud Platform, and then click **Import Data** in the upper-right corner.
+    1. In the TiDB Cloud Admin console, choose a target project and a target cluster deployed on the Google Cloud Platform. Then click **...** in the upper-right corner of the cluster area and select **Import Data**.
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md:108]:
```suggestion
    1. In the TiDB Cloud console, choose a target project and locate the target cluster deployed on the Google Cloud Platform. Then click **...** in the upper-right corner of the cluster area and select **Import Data**.
```

File: tidb-cloud/config-s3-and-gcs-access.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/config-s3-and-gcs-access.md:26]: Is this field updated for this feature?

File: tidb-cloud/config-s3-and-gcs-access.md (hunk 1 above)
Comment (TomShawn) [reply tidb-cloud/config-s3-and-gcs-access.md]: done

**Review by hfxsd — APPROVED** [review]
LGTM

File: tidb-cloud/migrate-data-into-tidb.md [hunk 5]
```diff
@@ -92,7 +92,7 @@ Depending on the location and size of your source data, the importing methods ar
 - If your source data is located in Amazon S3 cloud storage, take the following steps:
 
     1. Configure Amazon S3 access to allow TiDB cloud to access the source data in your Amazon S3 bucket. For more information, see [configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access). 
-    2. From the TiDB Cloud console, navigate to the TiDB Clusters page, and then click the name of your target cluster to go to its own overview page. In the upper-right corner, click **Import Data**, and then fill in the importing related information on the **Data Import Task** page.
+    2. From the TiDB Cloud console, navigate to the **Clusters** page, and then click the name of your target cluster to go to its own overview page. In the upper-right corner, click **...**, select **Import Data**, and then fill in the importing related information on the **Data Import** page.
```
Comment (qiancai) [inline tidb-cloud/migrate-data-into-tidb.md:95]:
```suggestion
    2. From the TiDB Cloud console, navigate to the **Clusters** page, and then click the name of your target cluster to go to its own overview page. In the **Import** area, click **Import Data**, and then fill in the importing related information on the **Data Import** page.
```

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**